pex/finders.py

```diff
@@ -250,14 +250,21 @@ def safer_name(name):
 
 
 def get_script_from_whl(name, dist):
-  # This is true as of at least wheel==0.24.  Might need to take into account the
-  # metadata version bundled with the wheel.
-  wheel_scripts_dir = '%s-%s.data/scripts' % (safer_name(dist.key), dist.version)
-  if dist.resource_isdir(wheel_scripts_dir) and name in dist.resource_listdir(wheel_scripts_dir):
-    script_path = os.path.join(wheel_scripts_dir, name)
-    return (
-        os.path.join(dist.egg_info, script_path),
-        dist.get_resource_string('', script_path).replace(b'\r\n', b'\n').replace(b'\r', b'\n'))
+  # This can get called in different contexts; in some, it looks for files in the 
+  # wheel archives being used to produce a pex; in others, it looks for files in the
+  # install wheel directory included in the pex. So we need to look at both locations.
```

Comment:
looks like this was the only caller of the `safer_name` function above, so would kill that along with the removal of it's usage.